<launch>
    <arg name="robot_mesh_path" default="$(find isaac_firefly_adaptation)/meshes/firefly.dae"/>
    <group ns="fake">
        <node pkg="collision_check" type="collision_check_node" name="collision_check" output="screen" respawn="true"/>
        <param name="robot_mesh_path" value="$(arg robot_mesh_path)"/>
    </group>

    <include file="$(find isaac_firefly_adaptation)/launch/demo.launch"/>
</launch>